Modified example to do recursive crawling in local mode

w0mbat opened the pull request.

jnioche commented:
I'd create a separate StatusUpdater for this instead of modifying StdOutStatusUpdater

jnioche commented on core/src/main/java/com/digitalpebble/storm/crawler/spout/RandomURLSpout.java:
```diff
@@ -42,15 +43,14 @@
-            "http://www.lemonde.fr/", "http://www.bbc.co.uk/",
-            "http://www.facebook.com/", "http://www.rmc.fr" };
-
-    public RandomURLSpout(String... urls) {
```
This constructor is very useful. It should be kept and populate the HashMap just like the default constructor does.

jnioche commented on core/src/main/resources/urlfilters.json:
```diff
@@ -4,7 +4,7 @@
       "name": "MaxDepthFilter",
       "params": {
-        "maxDepth": 0
+        "maxDepth": 10
```
That's unrelated to the functionality of this PR and can/should be overridden in the user config. It should not be changed here.

w0mbat commented:
@jnioche I updated my pull request. I added a separate topology, seperate configs and also documented the commands. This way it can run out-of-the-box.

Hope its better now :)

jnioche commented:
Duplicating the conf + topology + filters is unnecessary. See #224
